### ISSUE TITLE
"conda install requirements.txt" error

### ISSUE THREAD

**Issue by paul-bio** [eggnogdb/eggnog-mapper#363]
Hello.
I created my own environment via conda.
And followed from the beginning. 
And need help in installing requirements.txt by 
    conda install --file requirements.txt

those below are the error messages.
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed

ResolvePackageNotFound:
  - python=3.1

do i need to reinstall python with version 3.1?

**Comment by Cantalapiedra** [comment]
Hi @paul-bio ,

Did you try first with `conda install -c bioconda eggnog-mapper`?

Best,
Carlos

**Comment by paul-bio** [comment]
Yep. In my own conda environment, I installed eggnog-mapper,, 

But errors below emerged 
 ResovedPackageNotFound:

**Comment by Cantalapiedra** [comment]
Could you please specify the exact steps you followed?
From conda version and creating the environment, to install the package.
I will try to reproduce it.
Thanks

**Comment by paul-bio** [comment]
Thanks @Cantalapiedra.

1. $conda create -n emapper 
2. $conda activate emapper
3. $conda install -c bioconda eggnog-mapper
4. $conda install --file requirements.txt

And I got error on the 4th step, 

However, it seems like there were two more way to setup
So I tried  "$python setup.py install" instead of "$conda install --file requirements.txt".
And it seems working now.
should i replace setup process with "$python setup.py install"? with previous one?
